nix/static-binary.sh

```diff
@@ -43,12 +43,25 @@ while [ $# -ne 0 ]; do
       USE_HEAD="on"
       ;;
     --with-plugin=*)
-      plugins+=("${optarg}")
+      plugins+=("${optarg%/}")
```

Comment:
It would probably nice to add a comment documenting that this expects an array of local git repositories, with directory names corresponding to plugin names; it's not super-obvious from looking at the functions below.

(also, while writing this comment I noticed that `usage` in line 39 appears to be undefined.

Reply:
The `--with-plugin=` option can be given multiple times. I'll fix the usage part.